### PR TITLE
update FDA test docs

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
I have updated some of the testing documentation for Muons. 

*There is no associated issue.*
This does not require release notes, because it is developer docs.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_4_FDAG.rst [hunk 1]
```diff
@@ -22,45 +22,57 @@ Ensure that you have first run all the tests in group 2, then open the Frequency
...
 --------------
 - Change the drop-down menu at the top of the interface to "MaxEnt"
 - The interface should look different
 - Click the Calculate MaxEnt button 
-- This will be slow - a warning should be provided and the button disabled
-- When it is complete a workspace should appear "FFTMuon" and the button should be enabled 
-
+- The calculate button will be disabled and cancel enabled
+- Click the cancel button
+- Click Calculate
```
Comment (Harrietbrown) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_4_FDAG.rst:70]:
```suggestion
- Click Calculate MaxEnt
```

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_4_FDAG.rst [hunk 2]
```diff
@@ -22,45 +22,57 @@ Ensure that you have first run all the tests in group 2, then open the Frequency
...
+- The plot should update to a mostly flat line with a peak
+- Make sure everything in the top table is ticked
+- Click calculate
```
Comment (Harrietbrown) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_4_FDAG.rst:73]:
```suggestion
- Click calculate MaxEnt
```

**Review by Harrietbrown — CHANGES_REQUESTED** [review]
this does describe its current functioning, just some changes.

**Comment by AnthonyLim23** [comment]
thank you for spotting those

**Review by Harrietbrown — APPROVED** [review]
Good job, this should be good for the next round of manual testing.

**Pull request merged**